mypy/messages.py

```diff
@@ -153,8 +153,9 @@ def copy(self) -> 'MessageBuilder':
     def add_errors(self, messages: 'MessageBuilder') -> None:
         """Add errors in messages to this builder."""
         if self.disable_count <= 0:
-            for info in messages.errors.error_info:
-                self.errors.add_error_info(info)
+            for errs in messages.errors.error_info_map.values():
```

Comment:
Oh. This is where `error_info_map` is used. Though of course in this case the errors will never be flushed...